src/prognostic_equations/forcing/external_forcing.jl

```diff
@@ -136,8 +191,10 @@ function external_forcing_cache(Y, external_forcing::GCMForcing, params)
             set_insolation!(insolation)
             set_cos_zenith!(cos_zenith)
 
-            @. ᶜinv_τ_wind[colidx] = 1 / (6 * 3600)
-            @. ᶜinv_τ_scalar[colidx] = compute_gcm_driven_scalar_inv_τ(zc_gcm)
+            @. ᶜinv_τ_wind[colidx] =
+                compute_gcm_driven_momentum_inv_τ(zc_gcm; external_forcing_type)
+            @. ᶜinv_τ_scalar[colidx] =
```

Comment:
@Sbozzolo I realized that using @. with a dispatched function seems to result in an error, since it's trying to broadcast over the type as well (which does not have a length, so I get `no method matching length`). Am I missing something? I also tried making it a kwarg instead to no avail

Reply:
Add this line:
```
Base.broadcastable(x::AbstractGCMDrivenForcingType) = tuple(x)
```